<launch>
  <arg name="properties_file_" value="$(find cabin_vision)/cfg/camera.yaml" />
  <node name="object_deviation" pkg="cabin_vision" type="object_deviation" output="screen" >
    <!--param name="image_width" value="800" /-->
    <!--param name="image_height" value="600" /-->
    <param name="properties_file" type="string" value="$(arg properties_file_)" />
  </node>
</launch>
